src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/GroupsControllerTest.kt

```diff
@@ -72,39 +80,37 @@ class GroupsControllerTest {
   @Nested
   inner class `parent group` {
     @Test
-    fun create() {
+    fun create(): Unit = runBlocking {
       val childGroup = CreateGroup("CG", "Group")
       groupsController.createGroup(childGroup)
       verify(groupsService).createGroup(childGroup)
     }
-
     @Test
-    fun `create - group already exist exception`() {
+    fun `create - group already exist exception`(): Unit = runBlocking {
       doThrow(GroupExistsException("_code", "group code already exists")).whenever(groupsService)
         .createGroup(
           any()
         )
 
       @Suppress("ClassName") val Group = CreateGroup("_code", " group")
```

Comment:
Can we now remove  val Group = CreateGroup("_code", " group")

Reply:
TODO 21 - remove unused code (intelliJ already highlights that it's no longer used)